packages/lab/src/dialog/DialogContext.tsx

```diff
@@ -3,8 +3,11 @@ import { ValidationStatus } from "@salt-ds/core";
 
 export const DialogContext = createContext<{
   dialogId?: string;
-  status?: ValidationStatus;
-}>({});
+  status?: ValidationStatus | "";
```

Comment:
This doesn't make sense. We should use `undefined` to represent no status.

Reply:
assuming this would be the same for dialogId as well ?